checkstyle.xml

```diff
@@ -109,6 +109,9 @@
 		<module name="RedundantImport">
 			<property name="severity" value="error"/>
 		</module>
+		<module name="UnusedImports">
+			<property name="severity" value="error"/>
+		</module>
```

Comment:
You have squashed someone else's commits into your commit as well. While squashing make sure that you pick only your commits. If you used this command 'git reset --soft HEAD~X' make sure to squash first and then rebase. If you manually want to pick commits to squash try this http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git/5201642#5201642